doc/source/Resources/pyaedt_with_IDE.bat

```diff
@@ -111,10 +112,23 @@ if [%install_pyaedt%]==[y] (
         echo Installing Pyaedt from local wheels %arg1%
         pip install --no-cache-dir --no-index --find-links=%wheelpyaedt% pyaedt
     ) ELSE (
-        echo Installing Pyaedt from pip
+        IF EXIST %current_dir%.git (
+            echo Installing Pyaedt from local clone %current_dir%
```

Comment:
```suggestion
            echo Installing PyAEDT from local clone %current_dir%
```
Messages should have ending punction. I'm not sure where the period should go, so I didn't add it.